src/pages/projectSettings/tabs/EventLogTab/EventLogTab.test.tsx

```diff
@@ -0,0 +1,235 @@
+      },
+      perfEnabled: false,
+      notifyOnBuildFailure: false,
+      patchTriggerAliases: [],
+      githubTriggerAliases: [],
+      workstationConfig: {
+        gitClone: false,
+        setupCommands: null,
+        __typename: "WorkstationConfig",
+      },
+      triggers: [],
+      periodicBuilds: [],
+      tracksPushEvents: true,
+      hidden: false,
+      prTestingEnabled: true,
+      manualPrTestingEnabled: false,
+      githubChecksEnabled: false,
+      gitTagVersionsEnabled: true,
+      gitTagAuthorizedUsers: ["mohamed.khelif"],
+      gitTagAuthorizedTeams: ["evergreen"],
+      commitQueue: {
+        enabled: true,
+        requireSigned: false,
+        mergeMethod: "squash",
+        message: "",
+        __typename: "CommitQueueParams",
+      },
+    },
+    subscriptions: [],
+    vars: {
+      vars: {
+        node_path: "/opt/nodejs/node-v16.17.0-linux-x64/bin:$PATH",
+      },
+      privateVars: [],
+      adminOnlyVars: [],
+      __typename: "ProjectVars",
+    },
+    githubWebhooksEnabled: true,
+    __typename: "ProjectEventSettings",
+    aliases: [],
+  },
+  after: {
+    projectRef: {
+      identifier: "spruce",
+      repoRefId: "6352b7f70ae6065419d5a499",
+      enabled: true,
+      owner: "evergreen-ci",
+      repo: "spruce",
+      branch: "main",
+      displayName: "Spruce",
+      batchTime: 30,
+      remotePath: ".evergreen.yml",
+      spawnHostScriptPath: "",
+      dispatchingDisabled: false,
+      versionControlEnabled: false,
+      deactivatePrevious: true,
+      repotrackerDisabled: false,
+      stepbackDisabled: null,
+      patchingDisabled: false,
+      taskSync: {
+        configEnabled: false,
+        patchEnabled: false,
+        __typename: "TaskSyncOptions",
+      },
+      disabledStatsCache: false,
+      __typename: "Project",
+      private: false,
+      restricted: false,
+      admins: ["arjun.patel", "mohamed.khelif", "sophie.stadler"],
+      buildBaronSettings: {
+        ticketCreateProject: "EVG",
+        ticketSearchProjects: ["EVG"],
+        __typename: "BuildBaronSettings",
+      },
+      taskAnnotationSettings: {
+        jiraCustomFields: null,
+        fileTicketWebhook: {
+          endpoint: "",
+          secret: "",
+          __typename: "Webhook",
+        },
+        __typename: "TaskAnnotationSettings",
+      },
+      perfEnabled: false,
+      notifyOnBuildFailure: false,
+      patchTriggerAliases: [],
+      githubTriggerAliases: [],
+      workstationConfig: {
+        gitClone: false,
+        setupCommands: null,
+        __typename: "WorkstationConfig",
+      },
+      triggers: [],
+      periodicBuilds: [],
+      tracksPushEvents: true,
+      hidden: false,
+      prTestingEnabled: true,
+      manualPrTestingEnabled: false,
+      githubChecksEnabled: false,
+      gitTagVersionsEnabled: true,
+      gitTagAuthorizedUsers: ["mohamed.khelif"],
+      gitTagAuthorizedTeams: ["evergreen"],
+      commitQueue: {
```

Comment:
I think you might be missing this field for the test mocks!

<img width="510" alt="Screen Shot 2023-01-23 at 5 06 06 PM" src="https://user-images.githubusercontent.com/47064971/214161570-69d54d57-5fb1-47d2-b6ba-ee970db00a19.png">